### PR TITLE
Fixed the chapel mass driver computer being dense.

### LINKED ISSUE #15732: Invisible wall on box
### What is the problem?
An invisible wall in the chapel.
### Why is it a problem, or what should have happened?
No invisible wall.
### How can the problem be reproduced?
Start map, wall.

Wall
Invisible wall one block under me.

### PR DESCRIPTION
**Pull request by Buggy123**
Don't blame me for 70,000 line edits, blame broken map merge.

Fixes #15732

### CODE REVIEW

**Comment by Crushtoe** [comment]
It's not dense, it's just special.

**Comment by Buggy123** [comment]
Good point okay closing PR no problem here.

**Comment by Crushtoe** [comment]
Sarcasm is not allowed in coding. Ship him back to the factory, boys, he has defects.

**Comment by xxalpha** [comment]
Why does this require a map edit instead of changing the controls code so it is not dense?

**Comment by Buggy123** [comment]
Its a offshoot of the derelict's computer, which is dense.

**Comment by Jordie0608** [comment]
A change like this could be re-done via text editor and avoid the line diff

**Comment by Crushtoe** [comment]
Is it being dense why the tile to the left of it is impossible to walk on? I had thought it was to keep you from flying into space when you launch it if the windoor was left open.

**Comment by duncathan** [comment]
it's your fault you have 65000 line edits
use the proper map merge

**Comment by duncathan** [comment]
or just edit the file outside the map editor

**Comment by Buggy123** [comment]
Done.

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -15467,7 +15467,8 @@
 "aFA" = (
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
 /obj/machinery/computer/pod/old{
-	icon = 'icons/obj/airlock_machines.dmi';
+	density = 0;
+	 icon = 'icons/obj/airlock_machines.dmi';
```
Comment (KorPhaeron) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: Did you use spaces instead of tabs or is github just displaying these two lines strangely?

Comment (xxalpha) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: i fixed this in https://github.com/tgstation/-tg-station/pull/15761

**Comment by Buggy123** [comment]
I used tabs, i don't know why it's showing that.

**Comment by tkdrg** [comment]
You can try either deleting that one space or redoing this with the fixed map merger.

**Comment by Buggy123** [comment]
I think i broke it.

**Comment by Buggy123** [comment]
I think i fixed it.

**Pull request merged**